src/System.Diagnostics.TextWriterTraceListener/src/System/Diagnostics/TextWriterTraceListener.cs

```diff
@@ -122,6 +122,10 @@ public override void Close()
                 catch (ObjectDisposedException) { }
                 _writer = null;
             }
+
+            // We need to set the _fileName to null so that we stop tracing output, if we don't set it
```

Comment:
Just a bug you found during porting?

Reply:
Yes, found that you would Close the listener. But some time ago EnsureWriter was added and if _fileName is not null, it opens the stream using that _fileName, so people would be able to keep tracing events after closing the listener, contrary to the Close method description, which states, after closing, any trace should be a no-op.